### ISSUE TITLE
Bug: uniclip does not gracefully handle alternative forms (eg. an image) of clipboard content

### ISSUE THREAD

**Issue by awik32** [quackduck/uniclip#23]
**Description of the bug**
When a picture (and probably other forms of non-text data) gets added to the clipboard, uniclip hangs with the following output:
```
verbose: [sent: An error occurred wile getting the local clipboard]
error: [exit status 1]
error: [exit status 1]
error: [exit status 1]
...
error: [exit status 1]
^C

```

**Verbose output (Use the `--debug` option)**
See above.

**Steps to Reproduce**
For example, you might send a screenshot to the clipboard.

**Expected behavior**
Either print an error message and resume normal operations, *or* send the non-text data to the clients.  Perhaps this should be configurable.

Cheers,
Albert.

**Comment by quackduck** [comment]
What OS is this on? If linux, what clipboard utility are you using?

**Comment by awik32** [comment]
Yes, the server is Linux (Mint 21.2).  If I recall correctly, the clipboard utility is "xclip".

**Comment by frohro** [comment]
I too am having trouble with images.  I tried flameshot and the gnome screenshot tool using xsel.  I get text instead of the images.  Is there a way around this?  I have been using X11 and on Wayland when I copy an image, and paste it in the server it pastes "An error occurred wile getting the local clipboard."  This is on Ubuntu 24.04 (both computers).  Text works fine.

**Comment by momentmal** [comment]
are images (screenshots) even supported? would be a really nice feature to have.

**Comment by robina80** [comment]
i can copy/paste text but not images between different computers on the network

if you can sort this out it would be awesome

**Comment by ghost** [comment]
I get the spam with just text (arch here)

```
error: [exit status 1]

error: [exit status 1]
```

Server side I get

```
verbose: [sent: An error occurred wile getting the local clipboard]

verbose: [rcvd: An error occurred wile getting the local clipboard]

verbose: [sent: An error occurred wile getting the local clipboard]
```
